examples/02-plot/topo-map.py

```diff
@@ -1,4 +1,6 @@
 """
+.. _ref_topo_map_example:
```

Comment:
After working on this for a while, I've decided against it because it's become redundant especially when :ref:`ref_topo_map_example` is better represented as simply :ref:`topo_map_example`.  We can create another PR making this consistent.